tree/ntuple/v7/doc/specifications.md

```diff
@@ -395,6 +395,14 @@ If `flag==0x02` (_projected field_) is set,
 the field has been created as a virtual field from another, non-projected source field.
 If a projected field has attached columns,
 these columns are alias columns to physical columns attached to the source field.
+The following restrictions apply on field projections:
+  - The source field and the target field must have the same structure,
+    except for an `RNTupleCardinality` field that must have a collection field as a source.
+  - For streamer fields and leaf fields, the type name of the source field and the projected field must be identical.
+  - Projections involving variants or fixed-size arrays are unsupported.
+  - Projected fields must be on the same schema path of collection fields than the source field.
+    For instance, one can project a vector of struct with floats to individual vectors of floats but cannot
```

Comment:
```suggestion
    For instance, one can project a vector of structs with floats to individual vectors of floats but cannot
```
or alternatively, `vector of a struct`, but the suggestion is consistent with `vector of floats`.